subsys/bluetooth/controller/ll_sw/ull_llcp_internal.h

```diff
@@ -238,857 +238,220 @@ static inline bool is_instant_reached_or_passed(uint16_t instant, uint16_t event
 /*
  * LLCP Resource Management
  */
-bool ull_cp_priv_tx_alloc_is_available(void);
+bool llcp_tx_alloc_is_available(void);
+struct node_tx *llcp_tx_alloc(void);
 
-static inline bool tx_alloc_is_available(void)
-{
-	return ull_cp_priv_tx_alloc_is_available();
-}
-
-struct node_tx *ull_cp_priv_tx_alloc(void);
-
-static inline struct node_tx *tx_alloc(void)
-{
-	return ull_cp_priv_tx_alloc();
-}
-
-bool ull_cp_priv_ntf_alloc_is_available(void);
-
-static inline bool ntf_alloc_is_available(void)
-{
-	return ull_cp_priv_ntf_alloc_is_available();
-}
-
-struct node_rx_pdu *ull_cp_priv_ntf_alloc(void);
-
-static inline struct node_rx_pdu *ntf_alloc(void)
-{
-	return ull_cp_priv_ntf_alloc();
-}
-
-struct proc_ctx *ull_cp_priv_create_local_procedure(enum llcp_proc proc);
-
-static inline struct proc_ctx *create_local_procedure(enum llcp_proc proc)
-{
-	return ull_cp_priv_create_local_procedure(proc);
-}
-
-struct proc_ctx *ull_cp_priv_create_remote_procedure(enum llcp_proc proc);
-
-static inline struct proc_ctx *create_remote_procedure(enum llcp_proc proc)
-{
-	return ull_cp_priv_create_remote_procedure(proc);
-}
+bool llcp_ntf_alloc_is_available(void);
+struct node_rx_pdu *llcp_ntf_alloc(void);
 
+struct proc_ctx *llcp_create_local_procedure(enum llcp_proc proc);
+struct proc_ctx *llcp_create_remote_procedure(enum llcp_proc proc);
 
 /*
  * ULL -> LLL Interface
  */
-void ull_cp_priv_tx_enqueue(struct ll_conn *conn, struct node_tx *tx);
-
-static inline void tx_enqueue(struct ll_conn *conn, struct node_tx *tx)
-{
-	return ull_cp_priv_tx_enqueue(conn, tx);
-}
-
-void ull_cp_priv_tx_pause_data(struct ll_conn *conn);
-
-static inline void tx_pause_data(struct ll_conn *conn)
-{
-	return ull_cp_priv_tx_pause_data(conn);
-}
+void llcp_tx_enqueue(struct ll_conn *conn, struct node_tx *tx);
+void llcp_tx_pause_data(struct ll_conn *conn);
+void llcp_tx_resume_data(struct ll_conn *conn);
+void llcp_tx_flush(struct ll_conn *conn);
 
-void ull_cp_priv_tx_resume_data(struct ll_conn *conn);
-
-static inline void tx_resume_data(struct ll_conn *conn)
-{
-	return ull_cp_priv_tx_resume_data(conn);
-}
-
-void ull_cp_priv_tx_flush(struct ll_conn *conn);
-
-static inline void tx_flush(struct ll_conn *conn)
-{
-	return ull_cp_priv_tx_flush(conn);
-}
+/*
+ * LLCP Local Request
+ */
+struct proc_ctx *lr_peek(struct ll_conn *conn);
```

Comment:
Should lr_peek and rr_peek not also get the prefix llcp_? As they are not static to local/remote resp.

Reply:
Yes they should